cmake/app/boilerplate.cmake

```
@@ -346,6 +346,11 @@ foreach(root ${BOARD_ROOT})
         ${SHIELD_DIR_${s}}/dts_fixup.h
         )
 
+      list(APPEND
+        SHIELD_DIRS
+        ${shield_dir}/${s_dir}
```

This must be updated to:
```suggestion
        ${SHIELD_DIR_${s}}
```

---

@tejlmand Fixed, thanks.